whylog/config/__init__.py

```
@@ -125,8 +126,19 @@ def _load_rules(self):
         ]
 
     def _load_log_types(self):
-        #TODO fix log types loading
-        pass
+        matchers = defaultdict(list)
+        matcher_definitions = self._load_file_with_config(self._log_type_path)
+        for definition in matcher_definitions:
+            if definition['matcher_class_name'] == 'RegexFilenameMatcher':
+                matcher = RegexFilenameMatcherFactory.from_dao(definition)
+                matchers[definition['log_type_name']].append(matcher)
+            else:
+                #TODO: place for other filename matchers
```

`raise NotImplementedError()`, not omit some silently. Never fail silently.